src/components/Copy/Copy.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FC, useEffect, useRef, useState } from "react";
+
+import styles from "./Copy.module.scss";
+import { IProp } from "store/types";
+import { getPosition } from "helpers/getPosition";
+
+const Copy: FC<any> = ({ popupRef }: IProp) => {
+  const [pos, setPos] = useState({ currentTop: -390, currentLeft: 0 });
+  const divRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (popupRef) {
+      const { top, left } = popupRef.current!.getBoundingClientRect();
+      const { height, width } = divRef.current!.getBoundingClientRect();
```

Comment:
push those in helper too. And give popupRef and divRef as an  arguments to helper function